checkov/helm/runner.py

```diff
@@ -206,6 +207,8 @@ def convert_helm_to_k8s(self, root_folder: str, files: list[str], runner_filter:
         chart_directories = find_chart_directories(root_folder, files, runner_filter.excluded_paths)
         chart_dir_and_meta = list(parallel_runner.run_function(
             lambda cd: (cd, self.parse_helm_chart_details(cd)), chart_directories))
+        # remove parsing failures
+        chart_dir_and_meta = [chart_meta for chart_meta in chart_dir_and_meta if chart_meta[1]]
```

Comment:
Can we use a named tuple instead of accessing `chart_meta[1]`?
Meaning - unpack `chart_meta` into named components